### PR TITLE
rcon: Add Sunday coffee

### PR DESCRIPTION
**Pull request by jessealama**

### CODE REVIEW

File: rcon/2023/index.rkt [hunk 1]
```diff
@@ -489,9 +489,11 @@ My first Typed Racket program was an interpreter for the Lox language from Bob N
 @talk{Rhombus: Status update}
 ]
 
+  @coffee[@talk-time{Saturday, 10:30am}]
```
Comment (mflatt) [inline rcon/2023/index.rkt:492]: Sunday?

Comment (jessealama) [reply rcon/2023/index.rkt]: Oops, good catch. Fixed.

**Comment by jessealama** [comment]
@mflatt @rfindler How does this look?

**Comment by mflatt** [comment]
LGTM

**Comment by rfindler** [comment]
Me too.

Thanks!

**Comment by jessealama** [comment]
@mflatt @rfindler Thanks for taking a look! Could you push the button?

**Comment by mflatt** [comment]
Done!

**Pull request merged**